src/index.html

```diff
@@ -5,9 +5,48 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="stylesheet" href="./styles/style.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article class="card" data-qa="card">
+      <img
+        class="card__product card__product--image"
+        src="images/imac.jpeg"
+        alt="apple A1419 iMac"
+      >
+
+      <h2 class="card__product card__product--name">APPLE A1419 iMac 27" Retina <br>5K Monoblock (MNED2UA/A)</h2>
```

Comment:
card__product hasn't any sense, remove it from all class
```suggestion
      <h2 class="card__name">APPLE A1419 iMac 27" Retina <br>5K Monoblock (MNED2UA/A)</h2>
```